### DISCUSSION TITLE
Can't start the addon - can only run as pid 1

### DISCUSSION THREAD

**Discussion by yabbah** [tsightler/ring-mqtt#476] [Support Q&A] [answered]
I got this error message when try start the addon: s6-overlay-suexec: fatal: can only run as pid 1

Home assistant 2022.10.1
Supervisor 2022.09.1
Operating System 9.0

Any idea ?

**Comment by tsightler** [comment, marked as answer]
Are you sure you are running this project?  I've seen that error reported with the Ring Livestream addon, but that's from another project entirely.  Please provide a screenshot of the addon configuration page.

**Comment by yabbah** [comment]
So embarrassing. I'm sorry. I had the other one installed which I had forgotten. This addon works!
